tensorboard/pip_package/pip_smoke_test.sh

```diff
@@ -131,32 +131,51 @@ TMP_LOGDIR=$(mktemp -d --suffix _tensorboard_logdir)
 tensorboard --port="${TEST_PORT}" --logdir="${TMP_LOGDIR}" &
 TB_PID=$!
 
-TEST_URL="http://localhost:${TEST_PORT}/data/logdir"
 echo
 echo "tensorboard binary should be running at pid ${TB_PID}"
-echo "Sending test HTTP requests at URL: ${TEST_URL} (${NUM_RETRIES} retries)"
-echo
-
-RETRY_COUNTER=0
-REQUEST_SUCCEEDED=0
-while [[ "${RETRY_COUNTER}" -lt "${NUM_RETRIES}" ]]; do
-  sleep 1
-  STATUS_CODE=$(curl -Is "${TEST_URL}" | head -1 | cut -d ' ' -f 2)
-  if [[ "${STATUS_CODE}" == 200 ]]; then
-    echo
-    echo "Request to ${TEST_URL} succeeded (200)!"
-    echo
-    REQUEST_SUCCEEDED=1
-    break
-  else
-    : $(( RETRY_COUNTER++ ))
-    echo "Request to ${TEST_URL} failed. Will retry in 1 second..."
+echo
+
+function test_access_url(){
```

Comment:
Prefer `test_access_url() {` (no `function`). (`function` is a bash-ism; google-style permits you to use it, but only consistently, and it is not used elsewhere in this file (and omitting it is better than including it))